File: infra-market-server-vertx/src/main/kotlin/io/infra/market/vertx/config/RedisConfig.kt
package io.infra.market.vertx.config

import io.vertx.core.json.JsonObject

/**
 * Redis 配置
 */
data class RedisConfig(
    val host: String = "localhost",
    val port: Int = 6379
) {
    companion object {
        fun fromJson(json: JsonObject): RedisConfig {
            return RedisConfig(
                host = json.getString("host", "localhost"),
                port = json.getInteger("port", 6379)
            )
        }
    }
}

File: infra-market-server-dotnet/Config/RedisConfig.cs
namespace InfraMarket.Server.Config;

public class RedisConfig
{
    public string Host { get; set; } = "localhost";
    public string Port { get; set; } = "6379";
    public string Password { get; set; } = string.Empty;
    public int DB { get; set; } = 0;
}
